docs/platform-feature-parity.md

```diff
@@ -0,0 +1,243 @@
+* `n` - no current support (with page-link to heading underneath table, with details)
+* `` (blank) - not yet tried
+
+### Contributing
+
+The current integration test approach is only appropriate for validating `linux` binaries against `linux` OS/arch combinations.
+
+Validating `macOS` and `Windows` binaries requires adding coverage that runs on those platforms within Travis, but since Travis does not support containerised builds for either platform, assertions are limited to assert against the state of the CI hosts, where we're relying on that to predictable.
+
+## Matrix
+
+| Feature        | `linux` |        `macOS`         |        `Windows`         |
+|:---------------|:-------:|:----------------------:|:------------------------:|
+| **Assertions** |         |                        |                          |
+| `addr`         |   `f`   |                        |                          |
+| `command`      |   `f`   | [`p`](#command-macos)  |                          |
+| `dns`          |   `f`   |                        |                          |
+| `file`         |   `f`   |   [`p`](#file-macos)   |   [`p`](#file-windows)   |
+| `gossfile`     |   `f`   |                        |                          |
+| `group`        |   `f`   |                        |                          |
+| `http`         |   `f`   |                        |                          |
+| `interface`    |   `f`   |                        |                          |
+| `kernel-param` |   `f`   |                        |                          |
+| `mount`        |   `f`   |                        |                          |
+| `matching`     |   `f`   |                        |                          |
+| `package`      |   `f`   |                        |                          |
+| `port`         |   `f`   |                        |                          |
+| `process`      |   `f`   |                        |                          |
+| `service`      |   `f`   |                        |                          |
+| `user`         |   `f`   |                        |                          |
+| **Commands**   |         |                        |                          |
+| `add`          |   `f`   |                        |                          |
+| `autoadd`      |   `f`   |                        |                          |
+| `help`         |   `f`   |                        |                          |
+| `render`       |   `f`   |                        |                          |
+| `serve`        |   `f`   |  [`p`](#serve-macos)   |                          |
+| `validate`     |   `f`   | [`p`](#validate-macos) | [`p`](#validate-windows) |
+
+## Details
+
+Please keep this section sorted for ease of navigation.
+
+### Attribute: `addr`
+
+Not yet tested.
+
+#### `addr`: macOS
+
+#### `addr`: Windows
+
+### Attribute: `command`
+
+#### `command`: macOS
+
+Manually tested via [`serve`](#serve-macos).
```

Comment:
don't understand this? What works/doesn't work about command on OSX?

Reply:
That's a reference to later in the page where the serve command for macos runs a goss file that contains a command assertion which passes.